CODE_OF_CONDUCT.md

```diff
@@ -0,0 +1,83 @@
+## Goal
+
+Our goal is to provide a space where it is safe for everyone to contribute to
+open-source software in a respectful and peaceful manner.
+
+We value all contributions and want to make this organization and its surrounding community
+a place for everyone.
+
+As members, contributors and everyone else who may participate in the development, we strive
+to keep the entire experience civil.
+
+## Standards
+
+Our community is made for everyone to contribute to the codebase together.
+
+Our standards are:
+ - Do not harass, attack, or in any other way discriminate against anyone, including for their protected traits, including, but not limited to, sex, religion, race, appearance, gender, identity, nationality, sexuality, etc.
+ - Do not go off-topic, do not post spam.
+ - You are not required to like everybody, but you are required to treat everyone with respect.
+
+Examples of breaking each rule respectively include:
+ - Harassment, bullying or joking about someone's protected trait.
+ - Posting distasteful imagery, trolling, or political discussions.
```

Comment:
Also, pointed out on IRC: the kind of disruptive political discussions you may be thinking about falls under "on-topic" rules and the general rules regarding decorum and harassment, so this can of worms probably does not need to be here.